aten/src/ATen/native/quantized/cpu/qlinear.cpp

```diff
@@ -748,6 +922,11 @@ TORCH_LIBRARY_IMPL(_quantized, QuantizedCPU, m) {
   m.impl(TORCH_SELECTIVE_NAME("_quantized::linear"), TORCH_FN(QLinearInt8<false>::run));
 }
 
+TORCH_LIBRARY_IMPL(quantized, CPU, m) {
+  m.impl(TORCH_SELECTIVE_NAME("quantized::linear_fused_qdq_skip_quant"), TORCH_FN(QLinearInt8FusedQDQ<false>::run));
```

Comment:
So is this for qlinear_dynamic?

Reply:
Are we decomposing qlinear_dynamic into input_quant->qlinear?

If so note that this is not a quantized linear op or I dont know what to call this op because it outputs fp32 value.

Reply:
this is similar to qlinear_dynamic, except that the scale/zero_point is calculated statically instead of dynamically

Reply:
if by qlinear you mean static quantized linear, then this is not qlinear